### DISCUSSION TITLE
Day and time being changed to appointment

### DISCUSSION THREAD

**Discussion by virtualtom** [code4recovery/12-step-meeting-list#1542] [Q&A]
This happens after an import, the csv file has valid dates and times in it but sometimes it's random ones that change, and sometimes it's all meetings.  I can't find anything different in the import files, and this happened just today.
I'm on version 3.16, which updated 4 days ago, but today may be the first time our secretary did an import.  Therefore, it may be the version.

**Comment by joshreisner** [comment]
i'm able to export and re-import on our demo site without losing any meeting times. could you share the file you're using?

**Comment by virtualtom** [comment]
I downgraded to the last version and that fixed it.  This is the file I’m importing though￼

**Comment by joshreisner** [comment]
could you come to github and upload it? email attachments don't come through. thanks

**Comment by virtualtom** [comment]
Do you have a link?

What’s the url for the update site again, I need to do a refresh to get my meetings back in the app, convention weekend in Buffalo and people are freaking.

**Reply by joshreisner** [reply]
i just reloaded your data in meeting guide so that should be up to date now with what's on your site currently

**Comment by joshreisner** [comment]
the thread we're talking on is at https://github.com/code4recovery/12-step-meeting-list/discussions/1542

`https://github.com/code4recovery/12-step-meeting-list/discussions/1542`

thanks

**Comment by joshreisner** [comment]
ah great, thanks! i'm able to reproduce the problem on our end - looks like the affected days are the ones that have spaces before and/or after the day

<img width="540" alt="Screenshot 2024-10-18 at 7 08 25 PM" src="https://github.com/user-attachments/assets/6fdf8e9c-0576-44b8-974a-39b298515f7f">

we can release an update that trims these values and investigate if there was a behavior change on this recently

**Reply by virtualtom** [reply]
Thanks Josh!  I'll leave mine at the current level until that release.  I'll also have to figure out where I missed trimming those in my forms/DB.

**Reply by joshreisner** [reply]
great! we should have an update out tomorrow
